runtime/src/iree/hal/CMakeLists.txt

```diff
@@ -15,6 +15,27 @@ iree_cc_library(
     hal
   HDRS
     "api.h"
+    "allocator.h"
```

Comment:
not sure what you're doing here, but these are private files - _only_ api.h is public - so don't do this